pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -252,15 +252,15 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 	updateStats := true
 	summary, err := m.summaryProvider.Get(updateStats)
 	if err != nil {
-		klog.Errorf("eviction manager: failed to get summary stats: %v", err)
+		klog.ErrorS(err, "Eviction manager: failed to get summary stats")
 		return nil
 	}
 
 	if m.clock.Since(m.thresholdsLastUpdated) > notifierRefreshInterval {
 		m.thresholdsLastUpdated = m.clock.Now()
 		for _, notifier := range m.thresholdNotifiers {
 			if err := notifier.UpdateThreshold(summary); err != nil {
-				klog.Warningf("eviction manager: failed to update %s: %v", notifier.Description(), err)
+				klog.InfoS("Eviction manager: failed to update notifier", "notifier", notifier.Description(), "err", err)
```

Comment:
warning replaced by Informational?

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

Reply:
This is correct---"warning" log messages are being deprecated.